.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+Tensor Comprehensions Github Issues Guidelines
+----------------------------------------------
+
+If you have a feature request or a bug report (build issue), please open an issue on Github and fill the template below so we can help you better and faster. If you have some general
+questions about Tensor Comprehensions, please visit our [slack channel](https://tensorcomprehensions.herokuapp.com/) or email us at tensorcomp@fb.com
+
+For build issues, please add `[Build]` at the beginning of issue title.
+
+When submitting a bug report, please include the following information (where relevant):
+- OS:
+- How you installed TC (docker, conda, source):
+- Python version:
+- CUDA/cuDNN version:
+- Conda version (is using conda):
+- GCC/GXX version (if compiling from source):
+
```

Comment:
Add LLVM/Tapir version [or docker version]
Also add commit hash of our repo and submodules [if from source]

Ideally we make a script to generate all these versions and they copy output

Reply:
nice catch, updating